_posts/2020-06-16-aws-acronyms-overview.md

```diff
@@ -0,0 +1,154 @@
+﻿---
+title: "What is AWS? An Overview with Acronyms."
+categories: [craft]
+modified: 2020-06-18
+author: pratikdas
+excerpt: "AWS is vast with all kinds of services under different domains. Most AWS services are referred to by their acronyms in regular conversations. This article introduces the most commonly used AWS services along with their acronyms."
+image:
+  auto: 0028-stream
+---
+
+**AWS-Amazon Web Services provides a cloud computing platform comprising of services under a wide range of portfolios like compute, storage, networking, data, security, and many more.** This makes AWS vast and looks daunting to many newcomers. 
+
+I will attempt to make this easy here by describing a selection of AWS services along with their three-character acronyms. I have selected the services for this article, by thinking of the services required to build and run a customer-facing n-tier application.
```

Comment:
I have changed to "I intend to give you an overview of the most important AWS services accompanied by their three-character acronyms. I hope it serves as a valuable aid to begin the exploration of AWS. I have selected the AWS services by thinking of the services required to build and run a customer-facing n-tier application.
". I try to avoid using the word "high-level"